oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -109,6 +109,16 @@ public class ComputeEngineCredentials extends GoogleCredentials
   static final int MAX_COMPUTE_PING_TRIES = 3;
   static final int COMPUTE_PING_CONNECTION_TIMEOUT_MS = 500;
 
+  public enum Transport {
```

Comment:
I know the it is called `transport` in the token url, but since Transport in GAPIC has different meaning (the value of it is either gRPC or httpJson), can we name this differently? Maybe `AuthTransport`? Javadocs for what ALTS and MTLS mean would also be appreciated.

Reply:
Renamed to AuthTransport in latest commit and added some javadocs. Thanks!